document/Introduction.mdk

```diff
@@ -8,7 +8,7 @@ To form an overall picture of program quality, developers must often aggregate t
 produced by all of these tools.
 This aggregation is more difficult if each tool produces output in a different format. 
 
-This document defines a standard output format for static analysis tools.
+This document defines a standard output format for the output of static analysis tools.
```

Comment:
How does this redundancy help? perhaps 'standard format for the output of static analysis tools?